### PR TITLE
Info icon with tooltip mentioning advanced search features on tabbed search page

### PR DESCRIPTION
**Pull request by jpaddison3**
The relevant line is `"Quotes" and -minus signs are supported.`. PR best viewed with whitespace off.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203797018070141) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/search/SearchPageTabbed.tsx [hunk 1]
```diff
@@ -337,12 +351,20 @@ const SearchPageTabbed = ({classes}:{
...
+            {/* Ignored because SearchBox is incorrectly annotated as not taking null for its reset prop, when
+              * null is the only option that actually suppresses the extra X button.
+            // @ts-ignore */}
+            <SearchBox defaultRefinement={query.query} reset={null} focusShortcuts={[]} autoFocus={true} />
+          </div>
+          <LWTooltip
+            title={`"Quotes" and -minus signs are supported.`}
+            className={classes.searchHelp}
+          >
+            <InfoIcon />
```
Comment (s-cheng) [inline packages/lesswrong/components/search/SearchPageTabbed.tsx:366]: I think this icon should be lighter grey, plus prob smaller as well

File: packages/lesswrong/components/search/SearchPageTabbed.tsx [hunk 2]
```diff
@@ -121,6 +128,11 @@ const styles = (theme: ThemeType): JssStyles => ({
       ...theme.typography.body2,
     },
   },
+  searchHelp: {
+    [theme.breakpoints.down('sm')]: {
+      display: "none",
+    },
+  },
```
Comment (s-cheng) [inline packages/lesswrong/components/search/SearchPageTabbed.tsx:135]: I think this should be `xs`, or maybe even just visible on all screen sizes (not sure, but seems ok on small screens?)

File: packages/lesswrong/components/search/SearchPageTabbed.tsx [hunk 3]
```diff
@@ -79,11 +80,17 @@ const styles = (theme: ThemeType): JssStyles => ({
   searchIcon: {
     marginLeft: 12
   },
+  searchBoxRow: {
+    display: "flex",
+    alignItems: "center",
+    marginBottom: 15,
+    gap: '16px',
+  },
```
Comment (s-cheng) [inline packages/lesswrong/components/search/SearchPageTabbed.tsx:88]: Don't forget to move this over from `searchInputArea` as well:

```
[theme.breakpoints.down('xs')]: {
    marginBottom: 12,
},
```

**Review by s-cheng — APPROVED** [review]

File: packages/lesswrong/components/search/SearchPageTabbed.tsx (hunk 2 above)
Comment (jpaddison3) [reply packages/lesswrong/components/search/SearchPageTabbed.tsx:135]: The reason I set it so high is that it's a mouse-only feature. The tooltips are not a good experience on mobile.

**Pull request merged**